haystack/pipelines/base.py

```diff
@@ -426,21 +424,20 @@ def run(  # type: ignore
         debug: Optional[bool] = None,
     ):
         """
-        Runs the pipeline, one node at a time.
+        Runs the Pipeline, one node at a time.
 
-        :param query: The search query (for query pipelines only)
-        :param file_paths: The files to index (for indexing pipelines only)
-        :param labels:
-        :param documents:
-        :param meta:
+        :param query: The search query (for query pipelines only).
+        :param file_paths: The files to index (for indexing pipelines only).
+        :param labels: Ground-truth labels that you can use to perform an isolated evaluation of pipelines. These labels are input to nodes in the pipeline.
+        :param documents: A list of Document objects or a list of lists of Document objects to be processed by the Pipeline Nodes.
```

Comment:
`run` only takes a list of documents. (List of lists of documents only possible in `run_batch`.)